panther_driver/src/ClassicKinematics.py

```diff
@@ -1,13 +1,7 @@
 #!/usr/bin/python3
 
 import math
-import rospy
-from sensor_msgs.msg import BatteryState
-from sensor_msgs.msg import JointState
-from geometry_msgs.msg import Twist
-from geometry_msgs.msg import Pose
-from geometry_msgs.msg import TransformStamped
-import tf2_ros
+
 from PantherKinematics import PantherKinematics
 
 class PantherClassic(PantherKinematics):
```

Comment:
Shouldn't you also correct the calculations within the `inverseKinematics()` functions (for all kinematic types), so that you multiply the velocity with `dt_` instead of dividing by it in lines 39, 42 and 44?

Reply:
I completely forgot about this. Thanks for reminding me. It's fixed and tested already.